### ISSUE TITLE
Bugs with the death script

### ISSUE THREAD

**Issue by Helper2002** [ParkerBeck/Game-CSCI3308#1]
I downloaded the project with the death script commit. When I ran the demo, moving the player off the screen didn't reset the screen, and the console displayed warnings ("The referenced script on this behavior game object player is missing"). Is this intentional?

**Comment by S95F** [comment]
No, that shouldn't be happening. When I was running it it didn't display any warnings. There must have been something I missed when submitting that script.

**Comment by Helper2002** [comment]
That might be why the jumping animation isn't playing for me either. Was it playing for you when you made the player jump? I've been messing with the script to try and make it happen.

**Comment by vindication1118** [comment]
I haven't had the jump animation work either

**Comment by S95F** [comment]
It might just be the jump animation. The controller for that was primarily unity and not being run by any script. I'll try to find a way to make it work tomorrow. It all worked on my machine so I'm going to find what I need to do to upload a correct version. I'll try just uploading my version of the project.

**Comment by S95F** [comment]
Ok when i uploaded the rest of the project it looked like it changed something in the library folder which might have something to do with the animations.
